include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -405,22 +623,22 @@ template <typename _ExecutionPolicy, typename _Range>
 oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy>
 reverse(_ExecutionPolicy&& __exec, _Range&& __rng)
 {
-    auto __v = views::all(::std::forward<_Range>(__rng));
+    auto __v = views::all(std::forward<_Range>(__rng));
     auto __n = __v.size();
     auto __n_2 = __n / 2;
 
     auto __r1 = __v | views::take(__n_2);
     auto __r2 = __v | views::reverse | views::take(__n_2);
-    swap_ranges(::std::forward<_ExecutionPolicy>(__exec), __r1, __r2);
+    swap_ranges(std::forward<_ExecutionPolicy>(__exec), __r1, __r2);
```

Comment:
It is good point for the future. Now that place is not connected with Range productization, it remains in experimental namespace. I will redo that formal "::" change to avoid confusing.